sql/core/src/test/scala/org/apache/spark/sql/PlanStabilitySuite.scala

```diff
@@ -102,8 +104,8 @@ trait PlanStabilitySuite extends TPCDSBase with DisableAdaptiveExecutionSuite {
 
   private def isApproved(dir: File, actualSimplifiedPlan: String): Boolean = {
     val file = new File(dir, "simplified.txt")
-    val approved = FileUtils.readFileToString(file, StandardCharsets.UTF_8)
-    approved == actualSimplifiedPlan
+    val expected = FileUtils.readFileToString(file, StandardCharsets.UTF_8)
+    expected == actualSimplifiedPlan
```

Comment:
Address the comment from https://github.com/apache/spark/pull/29270#discussion_r476178381